README.md

```diff
@@ -1,3 +1,291 @@
+7. Interact with an instance of a custom class from another class file.
+8. Add a custom class as a property on another class, use the `self.` keyword to interact with it.
+
+## Class Libraries
+
+In Object-Oriented Programming (OOP), the library of available objects is as much a part of the language as the syntax itself. Without a library of objects to "instantiate" (i.e. to create on instance of), an object-oriented language would be like having grammar but no vocabulary—there exists the structure of a language, but nothing to actually communicate with.
+
+In Objective-C, individual libraries are called "frameworks". So far, you've worked exclusively with objects from the Core Foundation library. The vast majority of foundation framework classes are prepended with the `NS` prefix which stands for "NeXTSTEP". This was the computer product engineered by the "NeXT" company who developed the Core Foundation library in the late 1980s, building it with the [Objective-C][objective-c] language syntax created by [Brad Cox][brad_cox] in the early 1980s.
+
+[objective-c]: https://en.wikipedia.org/wiki/Objective-C
+[brad_cox]: https://en.wikipedia.org/wiki/Brad_Cox
+
+Because the number of objects available within the language as a whole is staggering, they are collected into separate libraries (or "frameworks") based upon their intended usages. Some of the commonly-used Apple frameworks that you'll see include:
+
+| Prefix | Framework       | Purpose |
+|:------:|:----------------|:--------|
+| `NS`   | Core Foundation | Basic Objective-C objects.
+| `UI`   | UI Kit          | "User Interface" objects for building views and buttons. |
+| `CA`   | Core Animation  | Drawing 2D animations based upon key frames. |
+| `CG`   | Core Graphics   | Rendering 2D graphic effects. |
+| `SK`   | Scene Kit       | Rendering 3D objects and effects. |
+| `CL`   | Core Location   | Handling location data on a GPS-capable device. |
+| `MK`   | Map Kit         | Integrating with Apple Maps. |
+
+Each of these libraries contains hundreds or even thousands of prefabricated class files. Because many of them are very large and have rather specific purposes, any library other than Core Foundation is not typically "imported" (or loaded) into a class file until the additional functionality of that supplemental library is required.
+
+### Custom Class Prefixing
+
+You may have noticed the `FIS` prefix on the `FISAppDelegate` class in previous labs; these are the initials for "**F**lat**i**ron **S**chool", and is the customized prefix that we use in all of our Objective-C labs. It's best practice to prefix custom classes in order to disambiguate them from the classes in other libraries. While it's unlikely that you'd create a custom class that coincides with a class in an Apple framework, name collisions become a very real possibility when importing third party frameworks found in the Open Source community (i.e. via CocoaPods like the Specta and Expecta testing frameworks).
```

Comment:
Mention that user generated content should have a three letter prefix. two letter is for apple things